web-common/src/features/metrics-views/workspace/editor/Placeholder.svelte

```diff
@@ -53,21 +53,20 @@
     const schemaResp = await queryClient.fetchQuery({
       queryKey: getConnectorServiceOLAPGetTableQueryKey({
         instanceId,
-        table: model.resource.model.state.table,
-        connector: model.resource.model.state.connector,
+        table: model?.resource?.model?.state?.table,
+        connector: model?.resource?.model?.state?.connector,
       }),
       queryFn: () =>
         connectorServiceOLAPGetTable({
           instanceId,
-          table: model.resource.model.state.table,
-          connector: model.resource.model.state.connector,
+          table: model?.resource?.model?.state?.table,
+          connector: model?.resource?.model?.state?.connector,
         }),
     });
 
-    const dashboardYAML = generateDashboardYAMLForModel(
-      modelName,
-      schemaResp.schema
-    );
+    const dashboardYAML = schemaResp?.schema
+      ? generateDashboardYAMLForModel(modelName, schemaResp?.schema)
+      : "";
```

Comment:
@ericpgreen2 @AdityaHegde -- please double check me here. Pretty sure that this would just fails silently previously if the schema was not defined (`generateDashboardYAMLForModel` does not handle that case),  but please verify that returning an empty string in case of no schema is ok